install/on-host-integration/kafka/install.yml

```diff
@@ -0,0 +1,14 @@
+id: kafka-integration-docs
+name: Kafka
+title: Kafka Monitoring Integration
+description: |
+  The New Relic Kafka on-host integration reports metrics and configuration data from your Kafka service. We instrument all the key elements of your cluster, including brokers (both ZooKeeper and Bootstrap), producers, consumers, and topics.
+
+target:
+  type: on-host-integration
+  destination: host
+
+install:
+  mode: link
```

Comment:
do we have a targetedInstall for Kafka?

Reply:
Unfortunately, no 😕 We set out to have one initially but the setup required a lot of variables set ahead of time, and we never progressed to adding it in.